Lesson4.ipynb

```diff
@@ -29,6 +33,1310 @@
     "\n",
     "Select **Lesson4.ipynb**"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Read csv file"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### What is csv file?\n",
+    "\n",
+    "A CSV is a comma separated values file which allows data to be saved in a table structured format. CSVs are a kind of spreadsheet same with Excel but with a .csv extension. Traditionally they take the form of a text file containing information separated by commas, hence the name.\n",
```

Comment:
Replace "CSVs are a kind of spreadsheet same with Excel but with a .csv extension." with
"A CSV file is similar to an Excel spreadsheet, though it doesn't have the style formatting and has a .csv extension instead."